### PR TITLE
warning if bootnodes but discovery disabled

### PR DESCRIPTION
**Pull request by macfarla**
Warn user if they have specified bootnodes but discovery disabled

Comes out with the other warnings:
```
2020-12-16 15:16:37.632+10:00 | main | WARN  | Besu | Discovery disabled: bootnodes will be ignored.
2020-12-16 15:16:37.640+10:00 | main | WARN  | Besu | Node permissioning config file set workdir/besu2/permissioning.toml but no permissions enabled
2020-12-16 15:16:37.640+10:00 | main | WARN  | Besu | Account permissioning config file set workdir/besu2/permissioning.toml but no permissions enabled
2020-12-16 15:16:37.640+10:00 | main | WARN  | Besu | Account permissioning smart contract address set 0x0000000000000000000000000000000000008888 but smart contract account permissioning is disabled.
```

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

**Review by rain-on — APPROVED** [review]
LGTM

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 1]
```diff
@@ -2261,7 +2261,9 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
 
     if (bootNodes != null) {
       try {
-
+        if (!peerDiscoveryEnabled) {
```
Comment (pinges) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2264]: We could move this block out of the try ...

**Review by pinges — APPROVED** [review]
just one comment :-)

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java (hunk 1 above)
Comment (macfarla) [reply besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java]: done

**Pull request merged**